recursion.js

```diff
@@ -0,0 +1,30 @@
+module.exports = function recursion(tree, lvl) {
```

Comment:
```suggestion
module.exports = function recursion(tree, lvl = 0l) {
```

after that you can delete
```js
const curLvl = lvl || 0; 
```

and use `lvl` instead